packages/gatsby-plugin-google-analytics/README.md

```diff
@@ -147,3 +149,35 @@ This plugin also supports several optional General fields documented in [Google
 - `transport`: string
 
 These fields can be specified in the plugin's `options` as shown in the [How to use](#how-to-use) section.
+
+## Troubleshooting
+
+### No actions are tracked
+
+#### Check the tracking ID
+
+Make sure you supplied the correct Google Analytics tracking ID. It should look like this: `trackingId: "UA-111111111-1"`
+
+#### Make sure plugin and script are loaded first
+
+The analytics script tag is not properly loaded into the DOM. You can fix this by moving the plugin to the top of your `gatsby-config.js` and into the head of the DOM:
+
+```javascript
+module.exports = {
+  siteMetadata: {
+    /* your metadata */
+  },
+  plugins: [
+    // Make sure this plugin is first in the array of plugins
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        trackingId: "UA-111111111-1",
+        head: true,
```

Comment:
Might be worth adding a note that this handles the issue on it's own. Want to avoid prompting other questions from the user as to whether they need to move the script to the head themselves.

Reply:
I added the note in the next commit =)